### DISCUSSION TITLE
Item Identification Plugin - Add an Ammo Category

### DISCUSSION THREAD

**Discussion by taylordhendricks** [runelite/runelite#15847] [Ideas]
The colour difference between Mithril, Adamant and Rune Arrows & Bolts is sometimes difficult to tell, add to this all the different tips you can put on each bolt and enchantments and I quickly lose track of what is where.

Can we add a new category for Ammo in the Item Identification and have it identify not only the metal of the arrow/bolt but also the tip type and enchantment/non-enchantment.

**Comment by Nightfirecat** [comment]
This would be fine, however these definitions would need to be written so they are immediately clear to users, and also not so long that the text overlaps into adjacent item slots.
